config/routes.rb

```diff
@@ -8,5 +8,7 @@
     delete "/publish-intent(/*base_path)", to: "publish_intents#destroy"
   end
 
+  post '/v2/content/:content_id/publish', to: 'content_items#publish'
```

Comment:
This should probably be inside the above `scope` block to disable the `.json` variant of this route.